README.md

```diff
@@ -27,6 +28,25 @@ This API is being used by 3 frontend projects currently being developed:
 - [Contact](#contact)
 - [License](#license)
 
+## Features
+
+- Once the App is installed user can view an onboarding screen which introduces the app and give user an idea of how it works.
```

Comment:
there are few bullet points here that are unrelated to what the backend code provides, like this line here. This is only related with the android repository. Can you please remove this, and other bullet points which are unrelated, since they might be only implemented on the frontend app implementations.